### PR TITLE
Laplan package

### PR DESCRIPTION
**Pull request by ian-r-rose**
Extracting utilites into a separately installable package. There are three submodules:

1. `laplan.zoning` for dealing with zoning data.
1. `laplan.pcts` for dealing with entitlements.
1. `laplan.census` for dealing with census data.

A general principle I'm trying to follow is that this package should be separately install-able, and should *not* require a connection to S3 or any database to function. So data ingest (via intake or otherwise) should mostly be separate.

One exception I am thinking of making to the above: the census data table is public, and while large, it is not *huge*. It might be worth including in the repo/package.

### CODE REVIEW

**Comment by tiffanychu90** [comment]
I see the Census functions as being applied to more Census tables than we downloaded. The tables can be broadly described as: 
1. count tables: counts for numerator and denominator. Ex: # households that fall into particular income range, as well as the total # households overall within a census tract (or any other geography)
1. percent tables: percent for numerator and count for denominator. Ex: 15 for people with less than HS education (which is 15%, not a count), and 1,000 households in census tract. These need to be converted to counts from percents.
1. dollar tables: median hh income, inflation-adjusted for each year. This is the median for all households or for a particular race/ethnicity. But, beware not to calculate summary statistics from these, only report values as is.

The tables we download fit into these, and can use any of the Census functions to clean and aggregate them. But, there are hundreds of other tables that we did not download that can also use any of the Census functions. As long as the user knows the type of table it is, they can use any of our functions to reshape.

**Comment by ian-r-rose** [comment]
> 
> 
> I see the Census functions as being applied to more Census tables than we downloaded. The tables can be broadly described as:
> 
>     1. count tables: counts for numerator and denominator. Ex: # households that fall into particular income range, as well as the total # households overall within a census tract (or any other geography)
> 
>     2. percent tables: percent for numerator and count for denominator. Ex: 15 for people with less than HS education (which is 15%, not a count), and 1,000 households in census tract. These need to be converted to counts from percents.
> 
>     3. dollar tables: median hh income, inflation-adjusted for each year. This is the median for all households or for a particular race/ethnicity. But, beware not to calculate summary statistics from these, only report values as is.
> 
> 
> The tables we download fit into these, and can use any of the Census functions to clean and aggregate them. But, there are hundreds of other tables that we did not download that can also use any of the Census functions. As long as the user knows the type of table it is, they can use any of our functions to reshape.

Makes sense. @tiffanychu90 can you summarize some of this info in the docstrings/readme of the package? Otherwise, I think this should be ready.

**Comment by tiffanychu90** [comment]
So much work, thank you @ian-r-rose! 

Ref: #49

**Comment by ian-r-rose** [comment]
🎉
